### PR TITLE
MissionController Home position unit testing

### PR DESCRIPTION
**Pull request by DonLakeFlyer**

### CODE REVIEW

**Comment by DonLakeFlyer** [comment]
@dagar Not sure what to do with this one? It just says Waiting for status to be reporting for travis. It should have kicked off a new build for my new commit. No Idea how I can kick this to do a new build?

**Comment by dagar** [comment]
Something seems broken on the travis side.
https://travis-ci.org/mavlink/qgroundcontrol/requests

File: .travis.yml [hunk 1]
```diff
@@ -89,7 +89,7 @@ script:
   - echo -en 'travis_fold:end:script.1\\r'
   - echo 'Running unittests' && echo -en 'travis_fold:start:script.2\\r'
   - if [[ "${TRAVIS_OS_NAME}" = "linux" && "${CONFIG}" = "debug" ]]; then ./debug/qgroundcontrol --unittest; fi
-  - if [[ "${TRAVIS_OS_NAME}" = "osx" && "${CONFIG}" = "debug" ]]; then ./debug/qgroundcontrol.app/Contents/MacOS/qgroundcontrol --unittest; fi
+- if [[ "${TRAVIS_OS_NAME}" = "osx" && "${CONFIG}" = "debug" ]]; then ./debug/qgroundcontrol.app/Contents/MacOS/qgroundcontrol --unittest:MissionManagerTest --logging:MissionManagerLog; fi
```
Comment (dagar) [inline .travis.yml]: This indentation might be the problem.

**Comment by DonLakeFlyer** [comment]
I think I fixed it and it's working now

**Pull request closed without merging**